modules/oadp-installing-oadp-rosa-sts.adoc

```diff
@@ -124,4 +124,8 @@ EOF
 The `enable` parameter of `restic` is set to `false` in this configuration because OADP does not support Restic in ROSA environments.
```

Comment:
The DPA block:
```
restic:
    enable: false
```
Is deprecated in OADP 1.3, we now use:
```
spec:  
  configuration:    
    nodeAgent:
      enable: true
      uploaderType: restic
```

Reply:
@sbeskin-redhat 
You added the "spec" twice, please remove.
You already have the spec define above.

Reply:
@sbahar619 
Done

Reply:
@sbeskin-redhat @sbahar619 I think we need to modify it a little bit as OADP 1.2 only has dpa.spec.configuration.restic field. As long as OADP 1.2 is supported we should not modify it. We can mention in point that if you are using OADP 1.3 then set NodeAgent.enable to false

Reply:
@PrasadJoshi12 @sbahar619 
Please take a look now.